### PR TITLE
docs: Add upgrade instructions to docs

### PR DESCRIPTION
**Pull request by eiddor**
Connects-to: #434
Change-type: patch
Signed-off-by: Roddie Hasan <roddie@krweb.net>

### CODE REVIEW

**Comment by eiddor** [comment]
@tmigone Am I missing something with this one, or is the "Balena build failed" something unrelated to the PR?  I've seen it a few times recently.

**Comment by tmigone** [comment]
@eiddor this looks good, sorry i took so long to review. I have just one suggestion which i will add now

File: docs/01-getting-started.md [hunk 1]
```diff
@@ -62,8 +62,19 @@ For example, if you don't plan on using the Spotify integration, you can edit th
...
+
+### Upgrading the application
+
+To deploy bug fixes or new features to your balenaSound application, the process is the same as a deployment: 
+
+- Install the [balena CLI tools](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md)
+- Login with `balena login`
+- Download this [project](https://github.com/balenalabs/balena-sound/) and from the project directory run `balena push <appName>` where `<appName>` is the name you gave your balenaCloud application during the provision step above. 
+
+> **Note:** If you receive the message "Application is ambiguous" during the push, then you will need to specify your balenaCloud username along with the `<appName>`, ie. `balena push <username>/<appName>`
```
Comment (tmigone) [inline docs/01-getting-started.md:77]: I would change the title of this section to just "Upgrade" and then have the subsection be "Upgrading via CLI". Then we can also add "Upgrading via one-click". Text for that could be:

```
To update your application via one-click deploy just click the Deploy with balena button below. Make sure you select your already existing balenaSound application so you don't create a new one!

[![balena deploy button](https://balena.io/deploy.svg)](https://dashboard.balena-cloud.com/deploy?repoUrl=https://github.com/balenalabs/balena-sound&defaultDeviceType=raspberry-pi)
```

Comment (tmigone) [reply docs/01-getting-started.md:77]: Let me know if i'm not being clear lol i need coffee...

**Review by tmigone — CHANGES_REQUESTED** [review]

File: docs/01-getting-started.md (hunk 1 above)
Comment (eiddor) [reply docs/01-getting-started.md:77]: Thanks @tmigone! I'll get these fixed this weekend (working remote right now).

**Pull request merged**